x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/SetUpgradeModeAction.java

```diff
@@ -28,9 +24,7 @@ private SetUpgradeModeAction() {
         super(NAME);
     }
 
-    public static class Request extends AcknowledgedRequest<Request> implements ToXContentObject {
-
-        private final boolean enabled;
+    public static class Request extends SetUpgradeModeActionRequest {
```

Comment:
Can you use `SetUpgradeModeActionRequest` directly here instead of extending it?

Reply:
No not if the serialised form has not change. Not super important, more of  nit